### PR TITLE
Add server host name configuration

### PR DESCRIPTION
**Pull request by GANGANI**
## Purpose
Document the server host name configuration.

Related PR: https://github.com/wso2/carbon-identity-framework/pull/2949
Resolves: https://github.com/wso2/product-is/issues/9886

### CODE REVIEW

File: en/docs/setup/changing-the-hostname.md [hunk 1]
```diff
@@ -10,7 +10,45 @@ Server.
...
+        ``` toml
+        [server]
+        hostname = "is.dev.wso2.com"
+        internal_hostname = "is.dev.wso2.com"
+        ``` 
+    
+        - **Approach 2**
+        
+        If you need to configure a different name for the `internal_hostname` from the `hostname` as follows. 
+        In this case you have to follow the same steps to generate the keystore for internal_hostname as well.
```
Comment (ayshsandu) [inline en/docs/setup/changing-the-hostname.md:44]: Can we configure multiple keystores for different hostnames in the same server?

File: en/docs/setup/changing-the-hostname.md [hunk 2]
```diff
@@ -10,7 +10,45 @@ Server.
...
+    
+    1.  Configuring internal host name
+    
+        If you expect to configure a server hostname for internal API calls, it can be configured as follows. 
+        This is an optional config with the default value `localhost`. This configuration will be utilized to build the 
+        internal absolute url of a service endpoint, that will be consumed at places that will generate internal API calls.
+            
+        ``` toml
+        [server]
+        internal_hostname = "internal.wso2.is"
```
Comment (ayshsandu) [inline en/docs/setup/changing-the-hostname.md:22]: Why does a user need to change the internal_hostname to a different one than "localhost"

File: en/docs/setup/changing-the-hostname.md (hunk 1 above)
Comment (GANGANI) [reply en/docs/setup/changing-the-hostname.md]: Updated the server hostname configuration steps with https://github.com/wso2/docs-is/pull/2279/commits/48e547f3b5a3873b2a9a71f7ae79662aca83c14d

File: en/docs/setup/changing-the-hostname.md (hunk 2 above)
Comment (GANGANI) [reply en/docs/setup/changing-the-hostname.md]: Updated the server hostname configuration steps with 48e547f

File: en/docs/setup/changing-the-hostname.md [hunk 3]
```diff
@@ -3,13 +3,34 @@
...
+1.  Change the hostname 
+    
+    Server hostname for internal API calls, is by default configured as `localhost`. This configuration is utilized 
+    to build the internal absolute url of a service endpoint, that will be consumed at places that will generate 
+    internal API calls. Hence, when configuring the hostname you can follow one from the below two options 
+    according to your requirement.
+
+    - **Option 1**
+
+    If you are configuring the hostnames in production for a proxied server, you need to configure
```
Comment (ayshsandu) [inline en/docs/setup/changing-the-hostname.md:15]: ...for a proxied server... -> ...for a load-balanced or proxied deployment...

**Review by ayshsandu — APPROVED** [review]

File: en/docs/setup/changing-the-hostname.md (hunk 3 above)
Comment (GANGANI) [reply en/docs/setup/changing-the-hostname.md]: Fixed https://github.com/wso2/docs-is/pull/2279/commits/965827548312dc99c576af89e7dea0fafcbdce7d

**Pull request merged**